src/Lykke.Service.HFT/Controllers/OrdersController.cs

```diff
@@ -68,7 +68,7 @@ public class OrdersController : Controller
 
             var clientId = User.GetUserId();
 
-            var orders = _orderStateCache.All().Where(x => x.ClientId == clientId); // todo: fix default limit of 101 element
+            var orders = await _orderStateCache.FilterAsync(x => x.ClientId == clientId); // todo: fix default limit of 101 element
```

Comment:
This change is not related to the task. Moreover it leads to client-side filtering. FilterAsync returns IEnumerable, not IQueryable.